public/src/components/drawer.tsx

```diff
@@ -210,6 +210,11 @@ export default function NavDrawer(): React.ReactElement {
             <span className={classes.super}>🎨</span>
           </ListItem>
         </Link>
+        <Link key="Audit Tests" to="/audit-tests" className={classes.link}>
+          <ListItem className={classes.listItem} button key="Audit Tests">
+            <ListItemText primary="Audit Tests" />
```

Comment:
I think "Test audits" would be a better name, since this is where we display the audit history of tests

Reply:
Done 👍🏼